### PR TITLE
[MRG+1] Add example use of axes.spines.SIDE prop in matplotlibrc

### PR DESCRIPTION
**Pull request by jni**
This addition is intended to prevent further questions about this on
the mailing list. ;)

https://mail.python.org/pipermail/matplotlib-users/2016-October/000669.html

### CODE REVIEW

**Comment by tacaswell** [comment]
Thanks @jni !

Documentation was missed as part of https://github.com/matplotlib/matplotlib/pull/4218

File: matplotlibrc.template [hunk 1]
```diff
@@ -322,6 +322,12 @@ backend      : $TEMPLATE_BACKEND
                                          # value of the data.
 
 
+# axes.spines.left   : True   # Turn off the top and right axes spines
+# axes.spines.bottom : True
+# axes.spines.top    : False
```
Comment (tacaswell) [inline matplotlibrc.template]: Can you make these 'True' (the default value)?

**Comment by NelleV** [comment]
Thanks @jni !

**Comment by NelleV** [comment]
backported as 4ea9892ea30

**Pull request merged**